cypress/integration/unit_tests/polygon_draw_test.js

```diff
@@ -176,6 +177,38 @@ describe('Unit test for ShapeData', () => {
     });
   });
 
+  it('Shows calculating before update finishes', () => {
+    const div = document.createElement('div');
+    document.getElementsByTagName('body')[0].appendChild(div);
```

Comment:
Does document.body.appendChild work here? It's what we're using in update_test and might be nice to keep consistent.